cine-flex/src/components/Button/Button.module.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../styles/color.module';
+
+.btn {
+    padding: 0.9375rem;
+    width: 15.625rem;
+    font-size: 1.5rem;
+    letter-spacing: 0.5px;
+    background: linear-gradient(to right, $start-color 0%, $mid-color 50%, $end-color 100%);
+    border: none;
+    text-transform: uppercase;
```

Comment:
could this be common in .btn class itself ?

Reply:
I don't get this @DurgaBNVL. Could you please elaborate this comment?